resources/views/layout/navigation.blade.php

```diff
@@ -84,27 +84,27 @@ class="fa-solid fa-xmark text-2xl text-brand"></i>
                 </button>
             </div>
             <button
-                class="w-full group hover:text-website-normal py-4 text-lg sm:text-xl text-start @if($current === "about") text-website-normal @endif"
+                class="w-full group hover:text-website-normal py-4 text-lg sm:text-xl text-start @if(Str::contains($current, 'about')) text-website-normal @endif"
                 onclick="window.location='{{ route("about") }}';">
                 <a class="my-4 p-2 start">{{ __("content.pages.about") }}</a>
```

Comment:
Why `a`? without `href`?

Reply:
I added `href` in button instead of `<a> `coz i wanted whole tile to be linkable, including text's background (which - in previous design was blue on hover), but there Is no need as I think now -  changed it